onnxruntime/contrib_ops/cuda/decoder/decoder_masked_self_attention.cc

```diff
@@ -51,7 +51,7 @@ Status DecoderMaskedSelfAttention<T1, T2>::ComputeInternal(OpKernelContext* cont
   const Tensor* cache_indir = context->Input<Tensor>(kCacheIndirectionInputIndex);
 
   auto& device_prop = GetDeviceProp();
-  DecoderMaskedSelfAttentionParams parameters;
+  DecoderMaskedMultiHeadAttentionParams parameters;
```

Comment:
Are we not supporting `relative_position_bias` via this kernel path ? If we do, we need to enhance the existing unit tests to include that input.

Reply:
There is no use case at the moment because the interface with combined past is only used in gpt2 now